### PR TITLE
Check Spark Max Init and Retry

### PR DESCRIPTION
**Pull request by balghane**
This PR adds some utilities from 3005 for initializing sparks and uses them on initializing sparks in all the subsystems. Additionally, it renames Calibrations to Cal because it's shorter.

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -51,13 +51,11 @@ public static final class SwerveModule {
     public static final double DRIVING_ENCODER_VELOCITY_FACTOR_METERS_PER_SECOND =
         ((WHEEL_DIAMETER_METERS * Math.PI) / DRIVING_MOTOR_REDUCTION) / 60.0; // meters per second
 
-    public static final double TURNING_ENCODER_POSITION_FACTOR_RADIANS = (2 * Math.PI); // radians
-    public static final double TURNING_ENCODER_VELOCITY_FACTOR_RAD_PER_SEC =
-        (2 * Math.PI) / 60.0; // radians per second
```
Comment (tails618) [inline src/main/java/frc/robot/Constants.java:56]: Why get rid of these? Are they incorrect and copied from rev?

File: src/main/java/frc/robot/Constants.java [hunk 2]
```diff
@@ -123,11 +121,7 @@ public static final class Lift {
   }
 
   public static final class Intake {
-    /* Scalar for deploy motor encoder from RPM to real degrees per seconds */
-    public static final double DEPLOY_MOTOR_ENCODER_VELOCITY_SCALAR = Constants.PLACEHOLDER_DOUBLE;
-
-    /* Scalar for arm motor encoder from rotations to real degrees */
-    public static final double DEPLOY_MOTOR_ENCODER_SCALAR = Constants.PLACEHOLDER_DOUBLE;
```
Comment (tails618) [inline src/main/java/frc/robot/Constants.java:130]: And these? Do we not need them?

File: src/main/java/frc/robot/Constants.java (hunk 1 above)
Comment (balghane) [reply src/main/java/frc/robot/Constants.java:56]: Correct but the utils copied in can already convert stuff to either degrees or radians so it's preferable to let the util do it

File: src/main/java/frc/robot/Constants.java (hunk 2 above)
Comment (balghane) [reply src/main/java/frc/robot/Constants.java:130]: Same as above

**Review by tails618 — APPROVED** [review]

**Pull request merged**